torch/utils/checkpoint.py

```diff
@@ -196,6 +199,18 @@ def checkpoint_sequential(functions, segments, *inputs, **kwargs):
     if kwargs:
         raise ValueError("Unexpected keyword arguments: " + ",".join(arg for arg in kwargs))
 
+    # To accept variadic arguments is not consistent with nn.Sequential.
+    # This interface will be changed at PyTorch 1.2.0.
+    # See also: https://github.com/pytorch/pytorch/issues/19260
+    if not inputs:
+        warnings.warn('no input to checkpoint_sequential has been deprecated, '
```

Comment:
"no input" -> "giving no inputs"
"will be raised since PyTorch 1.2.0" -> "will be raised since PyTorch 1.3.0" (we need atleast 1 stable version to deprecate things, right now it is 1.1.0)

Reply:
Great feedback and fix. I also capitalized the warning message for multiple inputs just like "Giving no inputs". Thank you so much!